src/components/MyKiva/BadgeTile.vue

```diff
@@ -104,11 +104,14 @@ const selectedTier = computed(() => {
 	const tiers = [];
 	tieredBadges.value.forEach(badge => {
 		const tier = badge.achievementData?.tiers?.find(t => !t.completedDate);
+		const tierBadgeData = getTierBadgeDataByLevel(badge, tier.level);
```

Comment:
@roger-in-kiva I'm currently seeing an error in development that I think is from this line:
```
TypeError: Cannot read properties of undefined (reading 'level')
    at ImpactDashboardPage-…Ew0aJ_.js:362:30356
    at Array.forEach (<anonymous>)
    at ImpactDashboardPage-…Ew0aJ_.js:362:30226
    at Zr.fn (main-8VcMS5f5.js:10:9412)
    at Zr.run (main-8VcMS5f5.js:10:1517)
    at get value (main-8VcMS5f5.js:10:9657)
    at HT (main-8VcMS5f5.js:10:1674)
    at get dirty (main-8VcMS5f5.js:10:1236)
    at y (main-8VcMS5f5.js:14:12520)
    at cp (main-8VcMS5f5.js:14:2693)
    at ne (main-8VcMS5f5.js:14:42805)
    at Zr.q [as fn] (main-8VcMS5f5.js:14:41794)
    at Zr.run (main-8VcMS5f5.js:10:1517)
    at G.w.update (main-8VcMS5f5.js:14:42660)
    at ie (main-8VcMS5f5.js:14:41531)
    at D (main-8VcMS5f5.js:14:41171)
    at p (main-8VcMS5f5.js:14:38057)
    at C (main-8VcMS5f5.js:14:40413)
    at x (main-8VcMS5f5.js:14:40961)
    at p (main-8VcMS5f5.js:14:37991)
    at C (main-8VcMS5f5.js:14:40413)
    at x (main-8VcMS5f5.js:14:40961)
    at p (main-8VcMS5f5.js:14:37991)
    at C (main-8VcMS5f5.js:14:40413)
    at I (main-8VcMS5f5.js:14:39760)
    at _ (main-8VcMS5f5.js:14:38656)
    at p (main-8VcMS5f5.js:14:38031)
    at Zr.q [as fn] (main-8VcMS5f5.js:14:41929)
    at Zr.run (main-8VcMS5f5.js:10:1517)
    at G.w.update (main-8VcMS5f5.js:14:42660)
    at An (main-8VcMS5f5.js:14:1381)
    at Ty (main-8VcMS5f5.js:14:3112)
```